src/content/docs/en/reference/container-reference.mdx

```diff
@@ -40,21 +40,21 @@ Enables rendering components using [HTML streaming](/en/guides/server-side-rende
 
 #### `renderers` option
 
-**Type:** `AstroRenderer[]`
+**Type:** `SSRLoadedRendere[]`
 
-A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script
+A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script.
```

Comment:
We've now lost the example that the second sentence describes, so people will have no reference how to do it the other way in "non-viteful environments". Did you mean to leave that earlier paragraph as is with no changes? Is that previous way still OK? If so, when/for what things?

Do we want to instead "switch" to only talking about the new way to load renderers, rather than make this an alternative?